### PR TITLE
clang-tools: update maintainers

### PR DESCRIPTION
**Pull request by Patryk27**
https://github.com/NixOS/nixpkgs/issues/146659#issuecomment-977794156

cc @aherrmann 🙂

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -8923,7 +8923,7 @@
     name = "pasqui23";
   };
   patryk27 = {
-    email = "wychowaniec.patryk@gmail.com";
+    email = "pwychowaniec@pm.me";
```
Comment (bjornfor) [inline maintainers/maintainer-list.nix:8926]: A separate commit please? (Same PR is fine.)

Comment (Patryk27) [reply maintainers/maintainer-list.nix:8926]: Sure, will do 🙂 Do we have a standardized format for commit messages related to such changes?

Comment (bjornfor) [reply maintainers/maintainer-list.nix:8926]: I don't know. But git log on the maintainers file will probably give you some good ideas.

Comment (Patryk27) [reply maintainers/maintainer-list.nix:8926]: kk, done 🙂

**Review by bjornfor — APPROVED** [review]

**Comment by aherrmann** [comment]
Thank you!

**Pull request merged**